domains/transfer/Testing/Testcontainers/PostgresContainer.cs

```diff
@@ -1,8 +1,7 @@
-using System.Threading.Tasks;
 using Testcontainers.PostgreSql;
 using Xunit;
 
-namespace Worker.IntegrationTests.Testcontainers;
+namespace Testing.Testcontainers;
 
```

Comment:
Why rename?

Reply:
Because I needed a shared testing project for claim automation integration tests and transfer automation integration tests